### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added an extra step to be on preview mode.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ You get a free version of Azure multi-factor authentication as part of your Offi
 
 All Office 2016 client applications support MFA through the use of the Active Directory Authentication Library (ADAL). This means that app passwords aren't required for Office 2016 clients. However, you need to make sure your Office 365 subscription is enabled for ADAL, or modern authentication.
 
-1. To enable modern authentication, go to the admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">https://admin.microsoft.com</a>.
+1. To enable modern authentication, go to the admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">https://admin.microsoft.com</a>. 
+2. Turn On the Preview Mode by Selecting Try the preview toggle located at the top of the Home page.
```
Comment (nenonix) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
2. Turn on preview mode by selecting **Try the preview** toggle located at the top of the Home page.
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 2]
```diff
@@ -33,7 +33,8 @@ You get a free version of Azure multi-factor authentication as part of your Offi
...
 2. Choose **Settings** \> **Services & add-ins** and then choose **Modern authentication** from the list.
```
Comment (nenonix) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
2. Choose **Settings** > **Services & add-ins** and then choose **Modern authentication** from the list.
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md (hunk 1 above)
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
2. Turn on the Preview Mode by selecting **Try the preview** toggle located at the top of the Home page.
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md (hunk 2 above)
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
3. Choose **Settings** \> **Services & add-ins** and then choose **Modern authentication** from the list.
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 3]
```diff
@@ -33,7 +33,8 @@ You get a free version of Azure multi-factor authentication as part of your Offi
...
 3. Check the **Enable modern authentication** box in the **Modern authentication** panel.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
4. Check the **Enable modern authentication** box in the **Modern authentication** panel. 
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md (hunk 1 above)
Comment (cmcatee-MSFT) [reply Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]: UI has changed. Step 2 should now read:
Turn on the new admin center by selecting the "Try the new admin center" toggle located at the top of the Home page.

Comment (cmcatee-MSFT) [reply Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]: Turn on the new admin center by selecting **Try the new admin center** toggle located at the top of the Home page.

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 4]
```diff
@@ -33,9 +33,10 @@ You get a free version of Azure multi-factor authentication as part of your Offi
 
 All Office 2016 client applications support MFA through the use of the Active Directory Authentication Library (ADAL). This means that app passwords aren't required for Office 2016 clients. However, you need to make sure your Office 365 subscription is enabled for ADAL, or modern authentication.
 
-1. To enable modern authentication, go to the admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">https://admin.microsoft.com</a>.
-2. Choose **Settings** \> **Services & add-ins** and then choose **Modern authentication** from the list.
-3. Check the **Enable modern authentication** box in the **Modern authentication** panel. 
+1. To enable modern authentication, go to the admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">https://admin.microsoft.com</a>. 
+2. Turn on the Preview Mode by selecting **Try the preview** toggle located at the top of the Home page.
```
Comment (mypil) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
2. Turn on the new admin center by selecting **Try the new admin center** toggle located at the top of the Home page.
```

**Pull request merged**